### PR TITLE
resolving conflicts from last branch

### PR DESCRIPTION
**Pull request by shodgetts**

### CODE REVIEW

File: src/components/RnR/RnR_sort_list.jsx [hunk 1]
```diff
@@ -11,27 +11,26 @@ class Sort extends Component {
...
-          if (response.status !== 200) { console.log('problem'); }
-          return response.json();
-        });
-      Promise.all([listData, metaData]).then((info) => { console.log(info); });
-    };
   }
 
   componentDidMount() {
-    this.getAllReviews();
+    const { productId, dispatch } = this.state;
```
Comment (jbc5099) [inline src/components/RnR/RnR_sort_list.jsx]: I would be pulling this from props and do away with state, unless there's an underlying issue with that that I'm not seeing.

File: src/tests/RnR_tests/RnR.test.jsx [hunk 2]
```diff
@@ -27,22 +27,24 @@ describe('List and sub components exist', () => {
...
-function testList() {
-  const props = {
-  };
-  const enzymeWrapper = shallow(<List />);
-  return {
-    props,
-    enzymeWrapper,
-  };
-}
+// Test no longer applicable
```
Comment (jbc5099) [inline src/tests/RnR_tests/RnR.test.jsx]: Unless you're saving this code to reuse later it should probably just be deleted.

Comment (jbc5099) [reply src/tests/RnR_tests/RnR.test.jsx]: same with the test lower down

File: src/components/RnR/RnR_sort_list.jsx (hunk 1 above)
Comment (jbc5099) [reply src/components/RnR/RnR_sort_list.jsx]: Or did this have something to do with that setTimeout you slacked me about?

**Comment by shodgetts** [comment]
@jbc5099, thanks for the suggestion - I'll definitely look into that for my next stage and get that refactored. Going to spend some time this afternoon on some refactoring and testing before rendering more of the reviews. 

Everything is working fine should is ready for merge

**Pull request merged**